app.mk

```diff
@@ -0,0 +1,203 @@
+#########################################################################
+# common include file for application Makefiles
+#
+# Makefile Common Usage:
+# > make
+# > make install
+# > make remove
+#
+# Makefile Less Common Usage:
+# > make art-opt
+# > make pkg
+# > make install_native
+# > make remove_native
+# > make tr
+#
+# By default, ZIP_EXCLUDE will exclude -x \*.pkg -x storeassets\* -x keys\* -x .\*
+# If you define ZIP_EXCLUDE in your Makefile, it will override the default setting.
+#
+# To exclude different files from being added to the zipfile during packaging
+# include a line like this:ZIP_EXCLUDE= -x keys\*
+# that will exclude any file who's name begins with 'keys'
+# to exclude using more than one pattern use additional '-x <pattern>' arguments
+# ZIP_EXCLUDE= -x \*.pkg -x storeassets\*
+#
+# Important Notes:
+# To use the "install" and "remove" targets to install your
+# application directly from the shell, you must do the following:
+#
+# 1) Make sure that you have the curl command line executable in your path
+# 2) Set the variable ROKU_DEV_TARGET in your environment to the IP
+#    address of your Roku box. (e.g. export ROKU_DEV_TARGET=192.168.1.1.
+#    Set in your this variable in your shell startup (e.g. .bashrc)
+##########################################################################
+DISTREL = ../dist
+COMMONREL = ../common
```

Comment:
I don't think it's kind for the Makefile is to create directories outside of the cloned project directory.

Two ways we can fix this:
1) remove all the '../' relative paths and make sure ZIP_EXCLUDE excludes them from the channel
2) Keep app.mk intact and replicate the Roku SDK layout by moving the example app into a subdirectory.  Then we could move generalUtils.brs,url.brs, and googleAnalytics.brs into a top level /common directory.  

I prefer 2 since it is how I, and possibly others, have all my channels laid out.